app/Commands/SshCommand.php

```diff
@@ -33,6 +34,14 @@ public function handle()
             ]);
         }
 
+        $name = $this->argument('site');
+
+        if (! is_null($name)) {
+            $answers = collect($this->forge->sites($this->currentServer()->id));
```

Comment:
answers?

Reply:
Oops…copied from elsewhere and forgot to change the name of the variable. Fixed now